src/data/markdown/translated-guides/en/02 Using k6/04 Thresholds.md

```diff
@@ -271,6 +264,23 @@ We have these thresholds:
 - A counter metric that keeps track of the total number of times content returned was **not** OK.
   The success criteria here implies that content can't have been bad more than 99 times.
 
+**⚠️ Bad practice.** Even though it is possible to define a threshold for the same metric more than once like this:
```

Comment:
This is not really a "bad practice", this is more of a "common mistake".

Reply:
And this sentence also doesn't seem right:
> Even though it is possible to define a threshold for the same metric more than once like this

It's not technically correct and I think we can use programming terms here, something like
> Since thresholds are defined as an object, it's not possible to specify different thresholds with the same key (metric name) like this:
> ... <example> ...
> Instead, specify them in the array as shown in https://k6.io/docs/using-k6/thresholds/#multiple-thresholds-on-a-single-metric

Reply:
General rule?

Reply:
general rule is even worse :sweat_smile: it's pretty much always a mistake

Reply:
@na-- not sure about this:

> Since thresholds are defined as an object, it's not possible to specify different thresholds with the same key (metric name) like this:

I meant that it is possible in the sense that k6 doesn't throw an error on it. And that's why one should pay attention in this case, to avoid hard to catch mistake. I.e. if I were to read "it's not possible to specify", I'd think that it'll be caught at the start and I don't need to worry about it but that's not the case.
Hm, maybe then:
> Even though it is possible to define a threshold for the same metric more than once in the script like this
> ...
> This is not possible in k6 engine since thresholds are defined as an object with unique keys (metric names). Instead, specify them in the array as shown in https://k6.io/docs/using-k6/thresholds/#multiple-thresholds-on-a-single-metric

?

Reply:
"though it is possible to define a threshold for the same metric more than once in the script like this" is false - it is not possible to do that in vanilla JavaScript. Yes, it's not a syntax error, but that doesn't mean it's possible. It's not a k6 thing, it's how JS works.

Reply:
Okay, I get your point about complete correctness; so what is your version then? If we take into account that it's best not to create a wrong impression for a reader :slightly_smiling_face:

Reply:
Here it is https://github.com/grafana/k6-docs/pull/514/files#r759222258

I think we shouldn't even give an example, just in case someone copies that without actually reading the text...